_posts/2021-07-7-bisq-vulnerability.md

```diff
@@ -4,18 +4,18 @@ title:  "Vulnerability in Bisq found and disclosed"
 excerpt: In July 2021 the Haveno team found and disclosed a critical vulnerability in Bisq
 ---
 
-In July 2021, while Haveno was in its infancies, we found a critical vulnerability in Bisq.
+In July 2021, while Haveno was in its infancy, we found a critical vulnerability in Bisq.
 
-This issue allowed an attacker to harvest user's payment information on Bisq. These information included:
+This issue allowed an attacker to harvest user payment information on Bisq. The information included:
 
 - Bank accounts
 - Full names
-- Potentially home addresses
+- (Potentially) Home addresses
 
-All these data could have been harvested at no cost at all for the attacker. We don't know for how long Bisq has had this vulnerability and if it was exploited. We know that it was probably not exploited at a large scale, or Bisq's support team would have noticed a spike in support tickets.
+All the data could have been harvested at no cost at all for the attacker. We don't know how long Bisq has had this vulnerability for and if it was exploited. We know that it was probably not exploited at a large scale, or Bisq's support team would have noticed a spike in support tickets.
 
-We reported the vulnerability to Bisq and helped them to patch it. They then released a patched version of their software, which fixed the problem (v1.7.0).
+We reported the vulnerability to Bisq and helped them patch it. They then released a patched version of their software, which fixed the problem (v1.7.0).
 
 The disclosure on Haveno's Twitter: [https://twitter.com/HavenoDEX/status/1412782658887766016](twitter.com/HavenoDEX/status/1412782658887766016).
 
-The disclosure on Bisq's Twitter: [https://twitter.com/bisq_network/status/1412536933910147072](twitter.com/bisq_network/status/1412536933910147072).
\ No newline at end of file
+The disclosure on Bisq's Twitter: [https://twitter.com/bisq_network/status/1412536933910147072](twitter.com/bisq_network/status/1412536933910147072).
```

Comment:
Noticing now that this link is wrong. Please change it to

```suggestion
The disclosure on Bisq's Twitter: [twitter.com/bisq_network/status/1412536933910147072](https://twitter.com/bisq_network/status/1412536933910147072).
```

Reply:
Alright, will do